src/Uno.UWP/Devices/Geolocation/Geolocator.Android.cs

```diff
@@ -125,9 +123,7 @@ public static Geoposition ToGeoPosition(this Location location)
 							Latitude = location.Latitude,
 							Longitude = location.Longitude,
 							Altitude = location.Altitude,
-						},
-						AltitudeReferenceSystem.Ellipsoid,
-						Wgs84SpatialReferenceId
```

Comment:
By removing this, are we losing some functionality (that we may not understand right now)?
Wouldn't it be better to fix the Geopoint ctor instead?

Reply:
We never had this functionality. It was added for wasm only by this commit. https://github.com/unoplatform/uno/commit/b7bba3d68e3ef597bedb8ebd892d2e4f14b059b5#diff-60dce75981e870b5855507ba793dbf87

These properties don't exist in UWP.

Reply:
But if this was added for WASM, are we breaking something by removing it?

Reply:
AltitudeReferenceSystem.Ellipsoid and Wgs84SpatialReferenceId is also in #2536.

Reply:
From the history, it looks like some changes that are relevant to WASM, were mistakenly also applied to Android. Maybe @MartinZikmund can confirm but this fix looks good to me.

Reply:
Correct (full) ToGeoPosition() method SHOULD put AltitudeReferenceSystem.Ellipsoid and Wgs84SpatialReferenceId values. Current implementation (at least, Android version) should be corrected, as in my PR #2536. Also, some other changes should be done, but this PR (2536) has small chances to be merged.

Reply:
@pkar70 thanks for the clarification, on closer inspection it looks like you're right, the better fix is to make the correct ctor available for Android.

Reply:
@pkar70 @davidjohnoliver OK, the changes we discussed have been made!